test/manual/amp-inline-gallery/rtl.amp.html

```diff
@@ -5,8 +5,8 @@
   <title>A Basic inline gallery</title>
   <link rel="canonical" href="amps.html" >
   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
-  <script async custom-element="amp-base-carousel" src="https://cdn.ampproject.org/v0/amp-base-carousel-0.1.js"></script>
-  <script async custom-element="amp-inline-gallery" src="https://cdn.ampproject.org/v0/amp-inline-gallery-0.1.js"></script>
+  <script async custom-element="amp-base-carousel" src="https://cdn.ampproject.org/v0/amp-base-carousel-1.0.js"></script>
```

Comment:
Should we be updating the versions here? Do any of the existing tests try to test 0.1 for this?

Reply:
`rtl.amp.html` for `amp-base-carousel` is the only one for the carousel components that have tests relying on the RTL sample file; the others were updated for the sake of manual testing / pr deploy bot. Happy to revert if you like but these changes should not break anything.

Reply:
I'm mostly just constantly confused about the versions. Maybe we should put 1.0 samples inside a 1.0 subfolder to avoid confusion?